### DISCUSSION TITLE
Wired vs Wireless ISP

### DISCUSSION THREAD

**Discussion by vvavepacket** [Chia-Network/chia-blockchain#5958] [Q&A]
Hello, multiple questions:

1.) What are the factors of winning a block in Chia?

I have 2 internet conenctions.

The first is wired and is very fast 50MBPS and has low latency. However it has disconnection once a day or so.

The second is wireless thru my mobile phone. Its very reliable (no disconnection) but has medium latency, and not so fast (3MBPS). However it does not have port 8444 open (behind some CGNAT), will this be a problem?

Im thinking of just using my mobile phone since its very reliable.. However Im concern about the latency... Lets say I use my mobile phone.. My port 8444 is blocked by my mobile provider, so does that mean I will have less chances of winning since I will be broadcasting my proof to the network, but they will not be able to contact me?

2. I guess do you have port 8444 to win a block, i thought 8444 is just for helping other peers to sync fatser since they can connect to you via 8444. but how about winning a block? Does it contribute?

3. Lets say there are 2 users who got a winning proof, and they broadcast to the network/peers.. Who decides who is the final winner? I assume its the one who broadcast it first and the one where it reaches to the wider amount of peers? Whats the tie breaker for this?

**Comment by tiny0little** [comment]
i think opened `tcp8444` is better, once-a-day-disconnects, latency and jitter not a problem
